### ISSUE TITLE
Some gallery images missing alt text

### ISSUE THREAD

**Issue by liaprins** [liaprins/liaprins.com#28]
- [ ] SoL iPad pairing: just copy from “3_ The Study of Life” in Notes, where I have all those images with my original plans for captions for them.
- [ ] Sketching for Ideas slides in Outcomes section: copy from Kirby version (ensure NO straight quotes or double straight quotes!!!!!)
- [ ] Data Vis guild: Networks in a Nutshell: I think can copy from Kirby version (ensure NO straight quotes or double straight quotes!!!!!)

**For all, ensure NO straight quotes or double straight quotes!!!!!**
